src/js/components/RangeInput/RangeInput.js

```diff
@@ -10,7 +10,10 @@ const RangeInput = forwardRef(
   ) => {
     const formContext = useContext(FormContext);
 
-    const [value, setValue] = formContext.useFormInput(name, valueProp);
+    const [num, setNum] = formContext.useFormInput({
```

Comment:
Why not keep this `[value, setValue]`?

Reply:
Ah I believe this is a mistake on my part when I was debugging the hook changes, will revert this back!